ui/src/app/views/concept-homepage.tsx

```diff
@@ -194,6 +213,29 @@ export const ConceptHomepage = withCurrentWorkspace()(
       }
     }
 
+    loadConceptDomainInfo(conceptDomainInfo) {
+      const conceptsCache: ConceptCacheItem[] = [];
+      const conceptDomainCounts: DomainCount[] = [];
+      conceptDomainInfo.items.forEach((domain) => {
```

Comment:
Rather than do this as a forEach, it may be worth doing these as two maps. (For simplicity reasons.)

That might make it feel more reasonable inlined.

Reply:
sure. This was pre existing i'll change this